src/ArtistDetail.js

```diff
@@ -0,0 +1,40 @@
+const artistsSeed = [
```

Comment:
When naming files, use [PascalCase](https://techterms.com/definition/pascalcase) only when the file exports a Class: eg: React components. In other cases you can use [camelCase](https://en.wikipedia.org/wiki/Camel_case).